src/snowflake/connector/encryption_util.py

```diff
@@ -63,7 +48,7 @@ def encrypt_stream(
         encryption_material: "SnowflakeFileEncryptionMaterial",
         src: IO[bytes],
         out: IO[bytes],
-        chunk_size: int = block_size * 4 * 1024,
+        chunk_size: int = 64 * megabytes,  # block_size * 4 * 1024,
```

Comment:
Shouldn't you use `kilobyte` here since you're replacing a 1024 constant?

Reply:
From benchmarks and testing different chunk sizes I found that more smaller reads lead to more lock contention and overall lower performance. I found that 64 MB worked well

Reply:
Oops, I meant kilobyte and not megabyte

Reply:
So this was meant to be just a simplification. I'll switch it to `kilobyte`